ReactiveFeedback/Feedback.swift

```diff
@@ -11,7 +11,8 @@ public struct Feedback<State, Event> {
     /// - parameters:
     ///   - events: The transform which derives a `Signal` of events from the
     ///             latest state.
-    public init(events: @escaping (Scheduler, Signal<State, Never>) -> Signal<Event, Never>) {
+
+    public init(events: @escaping (Scheduler, Signal<(State, Event?), Never>) -> Signal<Event, Never>) {
```

Comment:
isn't this a breaking change? it's not additive because signal type changes so all the operations on this signal will produce a tuple, not a single state

Reply:
^^

We should keep this initialiser as is.